src/engraving/libmscore/stafftype.h

```diff
@@ -422,7 +422,7 @@ class StaffType
     bool isDrumStaff() const { return _group == StaffGroup::PERCUSSION; }
     // static functions for font config files
     static std::vector<QString> fontNames(bool bDuration);
-    static bool fontData(bool bDuration, int nIdx, QString* pFamily, QString* pDisplayName, qreal* pSize, qreal* pYOff);
+    static bool fontData(bool bDuration, size_t nIdx, QString* pFamily, QString* pDisplayName, qreal* pSize, qreal* pYOff);
```

Comment:
Is nIdx fontIdx? Should be renamed to an explicit name

Reply:
Those variable names all use a prefix, b for bool, n for numeric, p for pointer, etc.
So changing just nIdx would certainly be wrong